cmd/olm/main.go

```diff
@@ -257,10 +258,29 @@ func main() {
 		go monitor.Run(op.Done())
 	}
 
+	// Emit CSV metric
+	if err = ensureCSVMetric(logger, op); err != nil {
```

Comment:
Why is this in `main` and not the operator package?

Reply:
+1 I have the same question.